index.bs

```diff
@@ -94,13 +101,13 @@ ISSUE: Should this specification also define a structural brand stricter than 'm
 
 <h3 id="image-item">AV1 Image Item</h3>
 
-<p>When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>, it is called an <dfn>AV1 Image Item</dfn>, and SHALL obey the following constraints:
-<ul>
-<li>The item data SHALL be the content of a single valid [=AV1 Sync Sample=] as defined in [[!AV1-ISOBMFF]] with the additional constraints that there SHALL be only one [=Sequence Header OBU=] and it SHALL have its <code>[=still_picture=]</code> flag set to 1 and SHOULD have its <code>[=reduced_still_picture_header=]</code> flag set to 1.
-<li>The item SHALL be associated with an [=AV1 Item Configuration Property=].</li>
-</ul>
-
-</p>
+When an image item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>, it is called an <dfn>AV1 Image Item</dfn>, and shall obey the following constraints:
```

Comment:
I tried to be consistent with the AV1-ISOBMFF spec where we use all capitals for normative words: SHOULD, SHALL,... This is also the way they are spelled in the conformance section https://aomediacodec.github.io/av1-avif/#conformance. Is this a problem?

Reply:
Yes, the conformance section is says that these words are supposed to be in lower case. I am quoting, "However, for readability, these words do not appear in all uppercase letters in this specification.". So, I am just trying to be consistent with what the spec already says.